schedule/kernel/kselftests.yaml

```diff
@@ -0,0 +1,8 @@
+name:          kselftests
+description:    >
+    Generic scheduler for Linux kselftests
+schedule:
+    - boot/boot_to_desktop
+    - kernel/kselftests
+    - kernel/kirk
+    - shutdown/shutdown
```

Comment:
I don't think either of these yaml files are needed, unless the kirk one is for testing kirk itself?

Reply:
We never know :)